config/routes.rb

```diff
@@ -221,6 +218,15 @@
 
   post :app_store_webhook, to: 'sync#sync_individual'
 
+  resource :errors, only: [] do
```

Comment:
There already an errors namespace, should these not go there?

Reply:
It's a bit tricky to merge a namespace with a resource - basically we have an errors module (the namespace) and an errors class (the resource). It'd be a worthwhile refactor to unify these at some point.